src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1055,7 +1055,8 @@ void Transformations::MainSnippets(void) {
     // To avoid performance degradations, we disable MHA tokenization into Subgraphs in LLMs'.
     // We consider the presence of `ScaledDotProductAttentionWithKVCache` and `PagedAttentionExtension` ops
     // in the model as a sign that this model is LLM.
-    const auto is_LLM = ov::op::util::has_op_with_type<intel_cpu::ScaledDotProductAttentionWithKVCache>(model) ||
+    const auto is_LLM = ov::op::util::is_large_language_model(model) ||
+                        ov::op::util::has_op_with_type<intel_cpu::ScaledDotProductAttentionWithKVCache>(model) ||
                         ov::op::util::has_op_with_type<ov::op::PagedAttentionExtension>(model);
```

Comment:
Do we still need these checks here?
If we do, can move them inside `is_large_language_model`? Since we created a dedicated helper for this check, all the logic related should be inside. What do you think?

Reply:
At first time I just wanted to align with GPU check - I was not sure that  PA should be in the check `is_llm` in GPU Plugin. But I found that they updated the condition included `PA`. So I moved `PA` to common helper leaving only `SDPAWithKVCache` because it's CPU-specific op.
Thanks!